### DISCUSSION TITLE
肝臟的用進廢退

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1563] [General]
一般來說腎臟的捐贈手術​
只需要住院五天等待傷口穩定​
就可以小心僅腎地回歸正常生活​
​
根據統計數據，只有 1% 的捐腎者​
會在術後感到不腎其擾、後悔當初決定​
若扣掉心理因素，不良影響其實微乎其微​
​
這其中的關鍵，是僅存的那粒腰子​
會為了幫離職同事擦屁股而變得爭強好腎​它不但會越長越大，能力也越來越 pro​

我覺得這樣的狀況就很像分組報告，當你發現你的隊友靠不住或是消失了，你不得不扛起來所有工作，讓你超越自己的極限
​

**Comment by zhiron544** [comment]
捐腎是一件偉大的事，但也需要有勇氣和決心。我欣賞作者的態度和風格，希望他能繼續保持健康和快樂。

**Comment by lin157644** [comment]
欸~標題是肝臟點進來是腎臟:smile:，我覺得捐贈任何器官都需要很大的勇氣，很難體會那種將自己身體的一部分奉獻出去的感受，沒想到對於腎臟負擔變大反而會更強健嗎，在親身體會一人扛分組報告幾次後後確實能力會變得扎實呢。
